hello.rb

```diff
@@ -234,7 +272,18 @@ def app_url
 # The scopes we are requesting (must match what we entered when
 # we registered the app)
 def scopes
-  'store_v2_products'
+  # Content   store_v2_content
+  # Customers store_v2_customers
+  # Geography default
+  # Marketing store_v2_marketing
+  # Products  store_v2_products
+  # Orders    store_v2_orders
+  # Payments  store_v2_information
+  # Shipping  store_v2_information
+  # Store     store_v2_information
+  # System    default
+  # Tax       store_v2_information
+  'store_v2_orders store_v2_products store_v2_customers store_v2_content store_v2_marketing'
```

Comment:
Similar to above, let's keep the comments here but change it back to `store_v2_products` since this is the only thing this sample app does at this time.